cmd/metamorph.go

```diff
@@ -358,17 +362,17 @@ func NewStore(dbMode string, folder string) (s store.MetamorphStore, err error)
 				return nil, err
 			}
 
-			ttlDays := viper.GetInt("metamorph.recordRetentionDays")
-			if ttlDays == 0 {
-				ttlDays = recordRetentionDaysDefault
+			dataRetentionDays, err := config.GetInt("metamorph.db.cleanData.recordRetentionDays")
+			if err != nil {
+				return nil, err
 			}
 
 			tableNameSuffix := viper.GetString("metamorph.db.dynamoDB.tableNameSuffix")
 
 			s, err = dynamodb.New(
 				awsdynamodb.NewFromConfig(cfg),
 				hostname,
-				time.Duration(ttlDays)*24*time.Hour,
```

Comment:
it does not have to be converted to time.Duration. You can just write 
`dataRetentionDays*24*time.Hour`. Seems much more readable to me.

Reply:
It seems the conversion is needed
```
go build ./...
# github.com/bitcoin-sv/arc/cmd
cmd/metamorph.go:375:5: invalid operation: dataRetentionDays * 24 * time.Hour (mismatched types int and "time".Duration)
```